### ISSUE TITLE
1.16.5 error when stopping the server

### ISSUE THREAD

**Issue by Chypik** [Multitallented/Civs#192]
I ask you not to close the topic at once in advance, I am writing through a translator. my English is bad

This server is running Paper version git-Paper-573 (MC: 1.16.5)
plugins:
Civs v1.9.1
PermissionsEx v1.23.4
Vault v1.7.3-b131
PhantomEconomy v1.9.30-RELEASE
ProtocolLib v4.6.0

error when stopping the server
```
[21:58:31] [Server thread/INFO]: Stopping the server
[21:58:32] [Server thread/INFO]: Stopping server
[21:58:32] [Server thread/INFO]: [Civs] Disabling Civs v1.9.1
[21:58:32] [Server thread/INFO]: [Civs] Civs Version: 1.9.1 is now disabled!
[21:58:32] [Server thread/ERROR]: Error occurred while disabling Civs v1.9.1 (Is it up to date?)
java.lang.NoClassDefFoundError: shaded/net/kyori/adventure/platform/bukkit/BukkitAudiences
	at org.redcastlemedia.multitallented.civs.chat.ChatManager.onDisable(ChatManager.java:88) ~[?:?]
	at org.redcastlemedia.multitallented.civs.Civs.onDisable(Civs.java:92) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:265) ~[patched_1.16.5.jar:git-Paper-573]
	at org.bukkit.plugin.java.JavaPluginLoader.disablePlugin(JavaPluginLoader.java:415) ~[patched_1.16.5.jar:git-Paper-573]
	at org.bukkit.plugin.SimplePluginManager.disablePlugin(SimplePluginManager.java:516) ~[patched_1.16.5.jar:git-Paper-573]
	at org.bukkit.plugin.SimplePluginManager.disablePlugins(SimplePluginManager.java:502) ~[patched_1.16.5.jar:git-Paper-573]
	at org.bukkit.plugin.SimplePluginManager.disablePlugins(SimplePluginManager.java:495) ~[patched_1.16.5.jar:git-Paper-573]
	at org.bukkit.craftbukkit.v1_16_R3.CraftServer.disablePlugins(CraftServer.java:435) ~[patched_1.16.5.jar:git-Paper-573]
	at net.minecraft.server.v1_16_R3.MinecraftServer.stop(MinecraftServer.java:887) ~[patched_1.16.5.jar:git-Paper-573]
	at net.minecraft.server.v1_16_R3.DedicatedServer.stop(DedicatedServer.java:745) ~[patched_1.16.5.jar:git-Paper-573]
	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:1163) ~[patched_1.16.5.jar:git-Paper-573]
	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:289) ~[patched_1.16.5.jar:git-Paper-573]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_252]
```

**Comment by Multitallented** [comment]
Should be fixed in v1.9.2

**Comment by Chypik** [comment]
decision: Civs v1.9.2
